packages/server-wallet/src/handlers/__test__/single-app-updater.test.ts

```diff
@@ -8,7 +8,8 @@ import {TestChannel} from '../../wallet/__test__/fixtures/test-channel';
 import {TestLedgerChannel} from '../../wallet/__test__/fixtures/test-ledger-channel';
 import {SingleAppUpdater} from '../single-app-updater';
 
-const testChan = TestChannel.create({channelNonce: 1});
+const FINAL = 8;
+const testChan = TestChannel.create({channelNonce: 1, finalFrom: FINAL});
```

Comment:
I think 
```suggestion
const finalFrom = 8;
const testChan = TestChannel.create({channelNonce: 1, finalFrom});
```
(and so on below) is clearer.

Reply:
I'm not sure about this.